packages/widgetbook_cli/bin/widgetbook_command_runner.dart

```diff
@@ -0,0 +1,107 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:args/command_runner.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:pub_updater/pub_updater.dart';
+
+import 'command/analyze.dart';
+import 'command/command.dart';
+import 'command/create.dart';
+import 'command/login.dart';
+import 'command/logout.dart';
+import 'command/publish.dart';
+import 'version.dart';
+
+/// The package name.
+const packageName = 'widgetbook_cli';
+
+class WidgetbookCommandRunner extends CommandRunner<int> {
+  WidgetbookCommandRunner({
+    Logger? logger,
```

Comment:
Do we need a license featuring Angelov in this file?

Reply:
I think so; most of the code is imported from the mason packages. Trying to figure out how I'll do the citation